lib/smart-survey-api.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const axios = require('axios');
+const querystring = require('querystring');
+const SmartSurveyClient = require('smartsurvey-client');
+
+module.exports = {
+  getResponses: (baseUrl, surveyID, endPoint, token, tokenSecret) => {
+    querystring.stringify({ foo: 'bar', baz: ['qux', 'quux'], corge: '' });
+    const apiUrlPart = querystring.stringify({'api_token': token, 'api_token_secret': tokenSecret});
+    axios.get(`${baseUrl}/${surveyID}/${endPoint}?${apiUrlPart}`)
```

Comment:
if you need to use that, then you can easily either augment/extend or fork+PR the request for that and still benefit from the smartsurveyclient's pagination, auth and other abstractions, rather than having to do that yourself